db/schema.rb

```diff
@@ -359,10 +359,10 @@
     t.integer "status", null: false
     t.string "currency", null: false
     t.string "name"
-    t.decimal "rate_amount", precision: 5, default: "0", null: false
-    t.decimal "credits_balance", precision: 5, default: "0", null: false
-    t.decimal "balance", precision: 5, default: "0", null: false
-    t.decimal "consumed_credits", precision: 5, default: "0", null: false
+    t.string "rate_amount", null: false
+    t.string "credits_balance", default: "0.00", null: false
+    t.string "balance", default: "0.00", null: false
+    t.string "consumed_credits", default: "0.00", null: false
```

Comment:
You need to reset your database to avoid these changes ;-)

Reply:
Fixed